### PR TITLE
Added The Missing Code in word2vec.ipynb

### PR DESCRIPTION
**Pull request by uzair-ali10**
Existing code did not load the TensorBoard notebook extension before using the embedded TensorBoard viewer in `site/en/tutorials/text/word2vec.ipynb`

### CODE REVIEW

**Comment by uzair-ali10** [comment]
Can anyone tell me why this CI (Continous Integration) failed??

File: site/en/tutorials/text/word2vec.ipynb [hunk 1]
```diff
@@ -269,7 +269,7 @@
       },
       "outputs": [],
       "source": [
-        "# Load the TensorBoard notebook extension\n",
```
Comment (8bitmp3) [inline site/en/tutorials/text/word2vec.ipynb]: The original was correct IMHO ("the... notebook extension")

**Comment by 8bitmp3** [comment]
> Can anyone tell me why this CI (Continous Integration) failed??

image

It's most likely because it's not passing the lint test.

```
Lint log:

[google::second_person] Prefer second person instead of first person. Found: we => you
```

Try this:

```diff
-         # We create a custom standardization function to lowercase the text and
+         # Create a custom standardization function to lowercase the text and
        # remove punctuation.
```

There's also a variable called `we` but I'm not 100% sure it's also causing the issue.

Hope this helps 👍

File: site/en/tutorials/text/word2vec.ipynb [hunk 2]
```diff
@@ -264,6 +264,17 @@
     {
       "cell_type": "code",
       "execution_count": null,
+      "metadata": {
+        "id": "10pyUMFkGKVQ"
+      },
+      "outputs": [],
+      "source": [
+        "# Load TensorBoard notebook extension\n",
```
Comment (8bitmp3) [inline site/en/tutorials/text/word2vec.ipynb:272]: Suggestion:

```suggestion
        "# Load the TensorBoard notebook extension\n",
```

File: site/en/tutorials/text/word2vec.ipynb (hunk 1 above)
Comment (uzair-ali10) [reply site/en/tutorials/text/word2vec.ipynb]: Thanks for the Fix!!!

**Comment by uzair-ali10** [comment]
@8bitmp3 

I still didn't get why CI is failing, it's because of some English grammatical mistake or what??

**Comment by 8bitmp3** [comment]
Try the possible solution by removing "we" in one of the cells under comments (as described above in https://github.com/tensorflow/docs/pull/1877#issuecomment-817299161)

**Comment by uzair-ali10** [comment]
@8bitmp3 
Well I did it still the problem persists :(

File: site/en/tutorials/text/word2vec.ipynb [hunk 3]
```diff
@@ -872,7 +872,7 @@
       },
       "outputs": [],
       "source": [
-        "# We create a custom standardization function to lowercase the text and\n",
+        "# create a custom standardization function to lowercase the text and\n",
```
Comment (8bitmp3) [inline site/en/tutorials/text/word2vec.ipynb:875]:
```suggestion
        "# Create a custom standardization function to lowercase the text and\n",
```

**Comment by uzair-ali10** [comment]
@8bitmp3 
Still the same :(

**Comment by 8bitmp3** [comment]
> is that variable `we` causing problem maybe??

Since `we` stands for "word embedding" (I think), you could try the following:

```diff
class Word2Vec(Model):
  def __init__(self, vocab_size, embedding_dim):
    super(Word2Vec, self).__init__()
    self.target_embedding = Embedding(vocab_size,
                                      embedding_dim,
                                      input_length=1,
                                      name="w2v_embedding")
    self.context_embedding = Embedding(vocab_size,
                                       embedding_dim,
                                       input_length=num_ns+1)
    self.dots = Dot(axes=(3, 2))
    self.flatten = Flatten()

  def call(self, pair):
    target, context = pair
-     we = self.target_embedding(target)
+     word_emb = self.target_embedding(target)
-     ce = self.context_embedding(context)
+     context_emb = self.context_embedding(context)
-     dots = self.dots([ce, we])
+     dots = self.dots([context_emb, word_emb])
    return self.flatten(dots)
```

And, after changing `# We create...` to `# Create... `, I ran nblint on the notebook (`python3 -m tensorflow_docs.tools.nblint word2vec.ipynb`) and the output showed that the `second_person` test was ✅:

```
...
Pass | google::second_person | Prefer second person instead of first person: https://developers.google.com/style/person
...
```

Hope this helps!

**Comment by uzair-ali10** [comment]
@8bitmp3 
Ohh!! Woww!! Thanks, worked like charm, now u may review the code.

**Review by MarkDaoust — APPROVED** [review]
Thanks.

**Pull request merged**